code/modules/research/designs/autolathe_designs.dm

```diff
@@ -423,7 +423,7 @@
 /datum/design/healthanalyzer
 	name = "Health Analyzer"
 	id = "healthanalyzer"
-	build_type = AUTOLATHE
+	build_type = AUTOLATHE | PROTOLATHE
```

Comment:
I suspect this will need a departmental flag so it doesn't start appearing in every single protolathe.

Reply:
You're absolutely right.